js/script.js

```diff
@@ -79,3 +80,52 @@ addBookForm.addEventListener('submit', (e) => {
     books.addBook(newBook);
   }
 });
+
+// DISPLAY DATE ON THE DOM
+/* function updateTime() {
+  setTimeout(() => {realTime()}, 1000);
+} */
+
+const realTime = () => {
+  const date = new Date();
+  const dateOptions = {
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+  };
+
+  const timeOptions = {
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: true,
+  };
+
+  const currentDate = date.toLocaleDateString('en-GB', dateOptions);
+  const currentTime = date.toLocaleTimeString('en-GB', timeOptions);
+  displayDate.innerHTML = `${currentDate} ${currentTime}`;
+
+  setTimeout(() => { realTime(); }, 1000);
+};
+
+realTime();
+
+const navLinks = Array.from(
+  document.body.querySelectorAll('header nav ul li a'),
+);
+
+const sections = Array.from(document.body.querySelectorAll('section'));
+
+navLinks.forEach((navLink) => {
+  navLink.addEventListener('click', () => {
+    const idForSectionToShow = navLink.getAttribute('href');
+    sections.forEach((section) => {
+      const id = section.getAttribute('id');
+      if (`#${id}` === idForSectionToShow) {
+        section.classList.remove('hidden');
+      } else {
+        section.classList.add('hidden');
```

Comment:
- [ ] [OPTIONAL] while adding a new book, when I click on the `add` book button, the book is submitted but the form input remains filled with the text I filled in. This really affects user interaction as they may not be aware if the data are submitted or not. **What  I would suggest as a solution is that you may add a form reset function inside the form submission event so that whenever the form data is submitted successfully, the form inputs are set to empty.**
💡 **HINT**: the method looks like 
```
form.reset()
```
where form is the variable referencing your form 😉